<launch>
    <!-- vim: set ft=xml noet : -->

    <arg name="transform_json_path" />
    <arg name="waypoint_json_path_1" />
    <arg name="waypoint_json_path_2" />
    <arg name="waypoint_json_path_3" />
    <arg name="waypoint_json_path_4" />
    <arg name="startup_delay" />
    <arg name="log_output" default="screen" />
    <arg name="respawn_conductor" default="false" />
    <arg name="is_test" default="false" />

    <node pkg="conductor" type="robocup" name="robocup_guided_node" required="$(eval not respawn_conductor)" clear_params="true" output="$(arg log_output)" respawn="$(arg respawn_conductor)">
        <param name="transform_json_path" type="string" value="$(arg transform_json_path)" />
        <param name="waypoint_json_path_1" type="string" value="$(arg waypoint_json_path_1)" />
        <param name="waypoint_json_path_2" type="string" value="$(arg waypoint_json_path_2)" />
        <param name="waypoint_json_path_3" type="string" value="$(arg waypoint_json_path_3)" />
        <param name="waypoint_json_path_4" type="string" value="$(arg waypoint_json_path_4)" />
        <param name="startup_delay" type="double" value="$(arg startup_delay)" />
        <param name="is_test" type="bool" value="$(arg is_test)" />

    </node>
</launch>
